packages/ui-library/src/components/transaction-confirmation/index.tsx

```diff
@@ -0,0 +1,374 @@
+            inflow === TransactionEventIncomingTypes.INCOMING
+              ? colors[mode].semantic.success.darker
+              : colors[mode].semantic.error.darker
+          }
+        >
+          {inflow === TransactionEventIncomingTypes.INCOMING ? '+' : '-'}
+          {amount.amountInUnits} {token.symbol}
+        </Typography>
+        {!!amount.amountInUSD && (
+          <Typography color={colors[mode].typography.typo3} variant="bodySmall">
+            ${amount.amountInUSD}
+          </Typography>
+        )}
+        {transferedTo && (
+          <Typography color={colors[mode].typography.typo3} variant="bodySmall">
+            <FormattedMessage
+              description="transactionConfirmationTransferTo"
+              defaultMessage="Transfered to: {account}"
+              values={{ account: `${transferedTo.slice(0, 6)}...${transferedTo.slice(-6)}` }}
+            />
+          </Typography>
+        )}
+      </StyledAmountContainer>
+    </StyledBalanceChange>
+    <Divider />
+  </>
+);
+
+interface SuccessTransactionConfirmationProps {
+  balanceChanges?: Omit<AmountBalanceChangeProps, 'mode'>[];
+  gasUsed?: Omit<GasBalanceChangeProps, 'mode'>;
+  successTitle?: React.ReactNode;
+  successSubtitle?: React.ReactNode;
+  onGoToEtherscan: () => void;
+  additionalActions: {
+    variant: ButtonProps['variant'];
+    color: ButtonProps['color'];
+    label: string;
+    onAction: () => void;
+  }[];
+  mode: 'light' | 'dark';
+}
+
+const SuccessTransactionConfirmation = ({
+  balanceChanges,
+  gasUsed,
+  successTitle,
+  additionalActions,
+  onGoToEtherscan,
+  mode,
+  successSubtitle,
+}: SuccessTransactionConfirmationProps) => {
+  return (
+    <>
+      <StyledConfirmationContainer>
+        <SuccessTickIcon size="100px" fontSize="inherit" />
+      </StyledConfirmationContainer>
+      <StyledTitleContainer>
+        <Typography variant="h5" fontWeight={700} color={colors[mode].typography.typo1}>
+          {successTitle}
+        </Typography>
+        {successSubtitle && (
+          <Typography variant="body" color={colors[mode].typography.typo2}>
+            {successSubtitle}
+          </Typography>
+        )}
+      </StyledTitleContainer>
+      <StyledBalanceChangesContainer>
+        {balanceChanges?.map((balanceChange) => (
+          <AmountBalanceChange mode={mode} key={balanceChange.token.address} {...balanceChange} />
+        ))}
+        {gasUsed && <GasBalanceChange mode={mode} {...gasUsed} />}
+      </StyledBalanceChangesContainer>
+      <StyledButonContainer>
+        {additionalActions.map((action) => (
+          <Button
+            variant={action.variant}
+            key={action.label}
+            color={action.color}
+            onClick={action.onAction}
+            fullWidth
+            size="large"
+          >
+            {action.label}
+          </Button>
+        ))}
+        <Button variant="outlined" color="primary" fullWidth onClick={onGoToEtherscan} size="large">
```

Comment:
how about showing the actual receipt component here?

Reply:
Oh good idea